### PR TITLE
add napari-locan-0.1.0

### PR DESCRIPTION
**Pull request by sdoose**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sdoose** [comment]
Hi @conda-forge/staged-recipes , this is ready for review!

File: recipes/napari-locan/meta.yaml
Comment (goanpeca) [inline recipes/napari-locan/meta.yaml:58]:
```suggestion
    - goanpeca
    - jaimrgp
    - sdoose
```

File: recipes/napari-locan/meta.yaml
Comment (goanpeca) [inline recipes/napari-locan/meta.yaml:44]:
```suggestion
  home: https://pypi.org/project/napari-locan
```

File: recipes/napari-locan/meta.yaml
Comment (goanpeca) [inline recipes/napari-locan/meta.yaml:13]:
```suggestion
```

File: recipes/napari-locan/meta.yaml
Comment (goanpeca) [inline recipes/napari-locan/meta.yaml:15]:
```suggestion
  number: 0
  noarch: python
```

File: recipes/napari-locan/meta.yaml
Comment (goanpeca) [inline recipes/napari-locan/meta.yaml:18]: Could we sort all the requirements alphabetically. It helps with maintenance and readability :)

File: recipes/napari-locan/meta.yaml
Comment (goanpeca) [inline recipes/napari-locan/meta.yaml:19]:
```suggestion
    - python >=3.9
```

File: recipes/napari-locan/meta.yaml
Comment (goanpeca) [inline recipes/napari-locan/meta.yaml:25]:
```suggestion
    - python >=3.9
```

File: recipes/napari-locan/meta.yaml
Comment (goanpeca) [inline recipes/napari-locan/meta.yaml:32]:
```suggestion
    - tomli >=1.1.0
```

Comment (goanpeca) [reply recipes/napari-locan/meta.yaml:32]: It is better to create a noarch package, if the only reason to not make it like that was adding this optional dependency, I suggest it is not worth it and we can leave it there as a dependency for all cases.

This will simplify packaging and produce less artifacts

**Review by goanpeca — APPROVED** [review]
.

**Review by goanpeca — CHANGES_REQUESTED** [review]
Hi @sdoose, thanks for the work on this recipe. The recipe looks good!

I left comments to improve it and make it even better :)

---

We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

That is why I added as maintainer. I agree to be added as maintainer

@jaimergp do you agree with being added as maintainers?

**Comment by sdoose** [comment]
Hi @goanpeca, thanks for your corrections. I believe this should be ready for review again.

**Review by goanpeca — APPROVED** [review]
Looks good @sdoose 🚀 

Thanks for working on this. Now waiting on @jaimergp approval to be a maintainer.

Cheers!

**Pull request merged**
